<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockMotionInterpolation</class>
 <widget class="QWidget" name="BlockMotionInterpolation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interpolation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DataComboBox" name="interpolationBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Estimation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="DataComboBox" name="estimationBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Algorithm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="DataComboBox" name="algorithmBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="sceneDetectionBox">
     <property name="text">
      <string>Detect scenes/shots</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>UI/datacombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
